typescript/src/deposit.ts

```diff
@@ -98,6 +98,19 @@ export type RevealedDeposit = Pick<
   treasuryFee: BigNumber
 }
 
+/**
+ * Represents an event emitted on deposit reveal to the on-chain bridge.fp
+ */
+export type DepositRevealedEvent = Pick<
+  Deposit,
+  "depositor" | "amount" | "blindingFactor" | "refundLocktime" | "vault"
+> & {
+  fundingTxHash: TransactionHash
+  fundingOutputIndex: number
+  walletPubKeyHash: string
+  refundPubKeyHash: string
+} & Event
```

Comment:
This can be simplfied:
```suggestion
export type DepositRevealedEvent = Deposit & {
  fundingTxHash: TransactionHash
  fundingOutputIndex: number
} & Event
```
Note that `*publicKeyHash` is widely used in the library. The contract uses an abbreviation (`*pubKeyHash`) but this can be easily mapped in `ethereum.ts` and not affect the library.

Reply:
d855431d690746b7b0e60ab57b12d376890010c7